aten/src/ATen/NamedTensorUtils.cpp

```diff
@@ -372,16 +372,16 @@ static std::vector<Dimname> compute_matmul_outnames(
 }
 
 void propagate_names_for_addmv(
-    TensorImpl* result,
-    TensorImpl* mat,
-    TensorImpl* vec,
-    TensorImpl* bias) {
-  if (!impl::has_names(result) && !impl::has_names(mat) &&
-      !impl::has_names(vec) && !impl::has_names(bias)) {
+    Tensor &result,
```

Comment:
IIUC, `TensorImpl*` is for legacy code and `Tensor &` is for new code. Am I correct? @zou3519 Could you take a look at if the change to this file makes sense?

Reply:
Yes, that is correct. Thanks for making this change.

(Also, based on the style in this file it should be `Tensor& result`, not `Tensor &result`)

Reply:
Fixed style, thanks for pointing out.